### ISSUE TITLE
logo for Amaze File Manager

### ISSUE THREAD

**Issue by damahboy** [TeamAmaze/AmazeFileManager#1174]
hii .. my graphic designer ,, my name is damahsyari, i intend to ask permission to give you a new logo for  Amaze File Manager

**Comment by EmmanuelMess** [comment]
You should post it as a picture here.

**Comment by VishalNehra** [comment]
@damahboy yes, please present with us some designs and if everything goes well, we shouldn't have any problems including them.

**Comment by damahboy** [comment]
![1](https://user-images.githubusercontent.com/38080324/38940714-10a8aa2e-4355-11e8-92be-41aa0c509b51.png)

if you do not like it i can redesign it

**Comment by EmmanuelMess** [comment]
It doesn't look like a file manager, it looks like some agenda o calendar.

**Comment by amirdesaingrafis** [comment]
Hello, I want to contribute to this awesome project, and I have a new logo design for you, if you allowed I will send to you :)

**Comment by EmmanuelMess** [comment]
Just add the image to your comment, no need for explicit authorization.

**Comment by amirdesaingrafis** [comment]
![1](https://user-images.githubusercontent.com/36655856/39630476-97a69000-4fd9-11e8-8ee4-75b3f241fd4e.png)

**Comment by amirdesaingrafis** [comment]
![3](https://user-images.githubusercontent.com/36655856/39631082-733287e0-4fdb-11e8-8de5-314443d971e9.png)

**Comment by amirdesaingrafis** [comment]
![2](https://user-images.githubusercontent.com/36655856/39631334-145c74c8-4fdc-11e8-88a4-2d0526877bfe.png)

**Comment by amirdesaingrafis** [comment]
Logo looks clean and easy to remember.
;)

**Comment by EmmanuelMess** [comment]
I like the proposals, especially because it seems easy to use as "multi shaped" icon. but the '+' inside the magnifying glass seems unnecessary.

**Comment by amirdesaingrafis** [comment]
i will edit, if everything is good i will send original file for you, svg,
png, eps, ai, cdr

Pada tanggal 5 Mei 2018 08.40, "Emmanuel" <notifications@github.com>
menulis:

**Comment by EmmanuelMess** [comment]
@VishalNehra what do you think?

**Comment by amirdesaingrafis** [comment]
![saasasas](https://user-images.githubusercontent.com/36655856/39667676-bb0f02a4-50e5-11e8-8496-e4cf0dd504c6.png)

**Comment by VishalNehra** [comment]
Wow, this new one is looking quite well. Very well done 👍 
But there needs to be something to distinguish it from the rest.. like in current icon we have stars. 
Think you can try putting on a few stars in magnifying glass and the folder? 
Ps. can you also try increasing the height of glass from the rest of the folder icon? Any visual way of showing it's above the folder?

**Comment by amirdesaingrafis** [comment]
![amaze file manager1](https://user-images.githubusercontent.com/36655856/39671140-ddb15530-513c-11e8-83d0-b12350e61976.png)

**Comment by amirdesaingrafis** [comment]
![2](https://user-images.githubusercontent.com/36655856/39671150-1230557c-513d-11e8-9929-b13702e6a702.png)

**Comment by amirdesaingrafis** [comment]
![3](https://user-images.githubusercontent.com/36655856/39671159-2ff7e58e-513d-11e8-9495-134976767d47.png)
![4](https://user-images.githubusercontent.com/36655856/39671160-36e99e64-513d-11e8-9569-fc450ed506ff.png)
![5](https://user-images.githubusercontent.com/36655856/39671162-3d6d358e-513d-11e8-85dd-70b7c2332d72.png)

**Comment by amirdesaingrafis** [comment]
![6](https://user-images.githubusercontent.com/36655856/39671165-52dfecfe-513d-11e8-8ed7-b3bb638b57b2.png)

**Comment by amirdesaingrafis** [comment]
![7](https://user-images.githubusercontent.com/36655856/39671168-668c99d2-513d-11e8-9c05-e2d5cdbb588d.png)
![8](https://user-images.githubusercontent.com/36655856/39671169-6eaca0a8-513d-11e8-8192-8cb4994f632e.png)
![9](https://user-images.githubusercontent.com/36655856/39671170-74a87c7a-513d-11e8-95a7-075a6ee23205.png)

**Comment by amirdesaingrafis** [comment]
@VishalNehra Thank you very much for your valuable comments. First, I want to say that you do not have to be an expert to criticize the proposed logo design, especially on your project. Of course you know the design logo is appropriate or does not work or reflect the function / meaning / purpose of the project.

If I Talk about design decisions, first of all I do not want to abandon the old logo design concept as in the image we currently have stars. To be able to use some stars, try with a soft splash with a magnifying glass. I think show the word "a" directly in the center of the magnifying glass icon. I would be those who would be easier to find files / icons that have the word prefix from the Amaze File Manager project. They will love it.

Lastly, I want to ask, what color would you like to change, let me know, I will do as soon as possible.

Have a nice day! smile

**Comment by EmmanuelMess** [comment]
Personally prefer [this](https://github.com/TeamAmaze/AmazeFileManager/issues/1174#issuecomment-386835907) one.

**Comment by amirdesaingrafis** [comment]
Thanks you @EmmanualMess. 
What do you think @VishalNehra...
are there any other suggestions.?😆

**Comment by VishalNehra** [comment]
I would like to comment on [this](https://user-images.githubusercontent.com/36655856/39671140-ddb15530-513c-11e8-83d0-b12350e61976.png) one.
It's really nice, however, the 'a' in the center isn't really catchy. I like the little circular trail left behind. So maybe a simple circle in between like suggested by @EmmanuelMess by with a similar trail left behind.
As for the trail of circles, I still think a trail of something similar to little stars in current icon would actually make sense.
And the voids left inside the trail (if zoomed in) doesn't really illustrate an illusion of 'depth' as seen in other material design guidelines. That's why I was pointing out that the magnifying glass circle should have a little more height than the rest of the icon and these voids should actually show there's an empty space between showing an illusion of depth.
These are my few thoughts, little tweaks and that would make it quite a good update to our current icon.
Again, thanks for all this work :)
Ps. I would like to ask @TranceLove  to shed his thoughts as he's also part of the team.

**Comment by amirdesaingrafis** [comment]
good comment. interesting discussion. I like your suggestion. you seem to understand more art than me. 😆 I will wait for further suggestions and feedback. I will edit again after an answer from @Trancelove.😊

**Comment by EmmanuelMess** [comment]
@VishalNehra I think only one thing can have pronounced shadow in the icon, be it the magnifying glass or the folder. Maybe the magnifying glass would be better? But as it is is very good.

**Comment by TranceLove** [comment]
I'd vote +1 for [this one](https://github.com/TeamAmaze/AmazeFileManager/issues/1174#issuecomment-386835907) too.

> @VishalNehra I think only one thing can have pronounced shadow in the icon, be it the magnifying glass or the folder. Maybe the magnifying glass would be better? But as it is is very good.

If you still want an A inside the folder, let's see if @amirdesaingrafis is clever enough to bring the shadow and the lens together to form an A. 😆

**Comment by amirdesaingrafis** [comment]
for this big project I am very interested to combine some inputs. I love this challenge. 😆 wait for me in a few moments.

**Comment by VishalNehra** [comment]
@EmmanuelMess not really. It (glass) can cast another relative shadow to it's surroundings showing it's above the folder too. As for the voids, it can leave a deeper shadow in them to show the illusion of depth. I would leave this to @amirdesaingrafis let's see what he comes up with (:

**Comment by amirdesaingrafis** [comment]
![01](https://user-images.githubusercontent.com/36655856/39691688-8fb3b626-5208-11e8-931c-3434886fb936.png)
![2](https://user-images.githubusercontent.com/36655856/39691706-9c3ae068-5208-11e8-9627-0835af79ba8a.png)

**Comment by amirdesaingrafis** [comment]
Icon Amaze File Manager
![3](https://user-images.githubusercontent.com/36655856/39691757-c5e89f7c-5208-11e8-9d4d-ba77d3ed7900.png)

**Comment by TranceLove** [comment]
Cool. I'm not sure if I'm appropriate to say the lens with the three dashes of shadows makes it looks like a bird... but it'd make an interesting effect IMO.

BTW @EmmanuelMess @VishalNehra I wonder if the light blue we chose currently for the default icon was too close to ES Explorer's blue?

**Comment by amirdesaingrafis** [comment]
the three lines left behind the lens symbolize the Amaze File Manager is a
faster application process. if the old logo has three irregular stars. New
logo mrmpunyai 3 lines that blend in the lens I'm sure this logo is
minimalist, clean and easy to remember. I myself like this one. 😊

Pada tanggal 7 Mei 2018 15.34, "Raymond Lai" <notifications@github.com>
menulis:

**Comment by EmmanuelMess** [comment]
[Nice](https://github.com/TeamAmaze/AmazeFileManager/issues/1174#issuecomment-386991581)!

**Comment by amirdesaingrafis** [comment]
is it final design logo / icon amaze file manager.?
if nothing else changes I will tilt the Original Files.

**Comment by EmmanuelMess** [comment]
@VishalNehra  please upload Play Store images to GitHub so we can change what's in Play Store.

**Comment by amirdesaingrafis** [comment]
Original File Amaze File Manager
https://drive.google.com/open?id=1O4DG-UsnaJAQ-iMU7P71dZDKNoyQSmFc

**Comment by VishalNehra** [comment]
Please also resize icons as per android launcher icon size guidelines.

**Comment by amirdesaingrafis** [comment]
Hi, @VishalNehra Launcher icons
https://drive.google.com/open?id=1rcD_pwlYYRTcmgMEOD4IlLdvxFPcuur9

**Comment by amirdesaingrafis** [comment]
is my logo accepted ...? :)

**Comment by EmmanuelMess** [comment]
@VishalNehra please add the Google Play resources to a folder.
